Makefile

```diff
@@ -90,6 +90,17 @@ GO_BUILD_HOOK = CGO_ENABLED=0 GOARCH=$(GOARCH) $(GO) -C contrib/rthooks/tetragon
 .PHONY: all
 all: tetragon-bpf tetragon tetra generate-flags test-compile tester-progs protoc-gen-go-tetragon tetragon-bench
 
+.PHONY: kind
```

Comment:
well this is a nit but I would have put that at the end of the Makefile instead of the beginning for clarity but this is okay...

Reply:
ok let me do it anyways